rs/registry/admin/src/main.rs

```diff
@@ -1254,6 +1260,88 @@ impl ProposalPayload<SubnetRentalRequest> for ProposeToRentSubnetCmd {
     }
 }
 
+/// Sub-command to submit a proposal to update the settings of a canister. When neigther
+/// `--controllers` nor `--no-controllers` is provided, the controllers will not be updated.
```

Comment:
"--no-controllers" is perhaps not clear enough IMO to prevent user error.

I think we want to say something like "--remove-all-controllers" so that it's not confused with "I don't want to update the controllers"

Reply:
Good idea. Done!